src/hooks/useBestTrade.ts

```diff
@@ -40,6 +40,7 @@ export function useBestTrade(
     useMemo(() => [amountSpecified, otherCurrency], [amountSpecified, otherCurrency]),
     useDebounceSwapQuoteFlag() === DebounceSwapQuoteVariant.Enabled ? DEBOUNCE_TIME_INCREASED : DEBOUNCE_TIME
```

Comment:
this is unrelated to your change but can we also move `useDebounceSwapQuoteFlag() === DebounceSwapQuoteVariant.Enabled` to its own `const` to avoid calling a hook inside of a hook?

Reply:
will do thats weird